src/plugins/intel_gpu/src/graph/include/program_node.h

```diff
@@ -167,6 +167,7 @@ struct program_node {
 
     program& get_program() { return myprog; }
     program& get_program() const { return myprog; }
+    std::shared_ptr<program> get_program_ptr() const { return myprog.shared_from_this(); }
```

Comment:
nit but ... program::ptr?

Reply:
I updated `std::shared_ptr<program>` to `program::ptr`. Thank you.